source/PlayerEntity.cpp

```diff
@@ -54,7 +54,7 @@ shared_ptr<Entity> PlayerEntity::create
 void PlayerEntity::init(AnyTableReader& propertyTable) {
    // Get values from Any
     Vector3 v;
-	float heading = m_headingRadians;
+	float heading = 0.0f;
```

Comment:
@bboudaoud-nv , this isn't a bug is it? I know there have been issues with getting the heading to load correctly from configs before.